components/match2/wikis/valorant/match_group_input_custom.lua

```diff
@@ -26,7 +26,7 @@ local MAX_NUM_PLAYERS = 10
 local MAX_NUM_MAPS = 9
 local MAX_NUM_ROUNDS = 24
 
-local _EPOCH_TIME_EXTENDED = '1970-01-01T00:00:00+00:00'
+local EPOCH_TIME_EXTENDED = '1970-01-01T00:00:00+00:00'
```

Comment:
don't do this change
it will cause merge conflicts in #3892 / #3897
either just let it be as is or use `DateExt.defaultDateTimeExtended` instead